paket.lock

```diff
@@ -0,0 +1,108 @@
+CONTENT: NONE
```

Comment:
It is the corresponding file for the paket.dependency. "paket install" will create it, but the build script assumes this already has been done.